### PR TITLE
added script

### PR DESCRIPTION
**Pull request by Tauqeer-Ahmed-99**
Added simple script for basic array filter and object practice.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  const filteredMovies = !filter
+    ? movies
+    : movies.filter((movie) => movie.title.includes(filter));
+
+  filteredMovies.forEach((movie) => {
+    const newMovieItem = document.createElement("li");
+    let text = movie.title + " - ";
+    for (const key in movie) {
+      if (key !== "title" && key !== "id") {
+        text = text + `${key}: ${movie[key]}`;
```
[the remark this answers is not in the document]
Comment (Tauqeer-Ahmed-99) [reply index.js]: Changed code as per your suggestions @AlishaPrasad  Ma'am.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    id: Math.random(),
+    title: titleValue,
+    [extraNameValue]: extraValueValue,
+  };
+
+  movies.push(movie);
+  renderMovies();
+}
+
+function renderMovies(filter = "") {
```
Comment (faran4engg) [inline index.js:25]: What's this feature `filter = ""`
Can you name it?

File: index.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  const movieList = document.getElementById("movie-list");
+
+  if (movies.length === 0) {
```
Comment (faran4engg) [inline index.js:28]: `!movies.length`

Comment (Tauqeer-Ahmed-99) [reply index.js:28]: Yes changing it rn

Comment (Tauqeer-Ahmed-99) [reply index.js]: changed it!
@faran4engg

**Review by faran4engg — APPROVED** [review]

**Pull request merged**